### DISCUSSION TITLE
prisma.errors.DataError: Error opening a TLS connection: error:0A000127:SSL routines:tls_construct_ctos_supported_groups:no suitable groups:ssl/statem/extensions_clnt.c:238:No groups enabled for max supported SSL/TLS version

### DISCUSSION THREAD

**Discussion by litagent** [prisma/prisma#22702] [Q&A]
### Question

prisma.errors.DataError: Error opening a TLS connection: error:0A000127:SSL routines:tls_construct_ctos_supported_groups:no suitable groups:ssl/statem/extensions_clnt.c:238:No groups enabled for max supported SSL/TLS version

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by litagent** [comment]
prisma = "0.9.1"

**Reply by ludralph** [reply]
Hi @litagent 👋 

Thank you for raising this question.

Do you imply that you are using Prisma version 0.9.1?  Also, the error message looks like an issue with the TLS connection setup, possibly related to the SSL/TLS configuration.

Can you please tell us more about your project setup?

**Reply by litagent** [reply]
Hi @ludralph , That issue got resolved when i disable TLS.
Now, i'm getting a new issue:

httpx.ConnectError: All connection attempts failed
2024-01-25 12:09:56.267	
ERROR:  Exception in ASGI application
2024-01-25 12:09:53.988	
httpx.ConnectError: All connection attempts failed

2024-01-25 12:09:53.988	
    raise mapped_exc(message) from exc
2024-01-25 12:09:53.988	
  File "/app/.venv/lib/python3.11/site-packages/httpx/_transports/default.py", line 83, in map_httpcore_exceptions
2024-01-25 12:09:53.988	
    self.gen.throw(typ, value, traceback)
2024-01-25 12:09:53.988	
  File "/usr/local/lib/python3.11/contextlib.py", line 158, in __exit__
2024-01-25 12:09:53.988	
    with map_httpcore_exceptions():
2024-01-25 12:09:53.988	
  File "/app/.venv/lib/python3.11/site-packages/httpx/_transports/default.py", line 365, in handle_async_request
2024-01-25 12:09:53.988	
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-01-25 12:09:53.988	
    response = await transport.handle_async_request(request)
2024-01-25 12:09:53.988	
  File "/app/.venv/lib/python3.11/site-packages/httpx/_client.py", line 1719, in _send_single_request

**Reply by litagent** [reply]
i'm setting up superagent projects in local, which use prisma.

**Reply by ludralph** [reply]
Hi @litagent 

Are you using Prisma with Python? If yes, it would be best if you could open an issue in the Prisma Python [repository](https://github.com/RobertCraigie/prisma-client-py).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
